Automattic/wp-calypso
# Jetpack Cloud: Remove notices from the Current Site

Currently we load different notices async in the Current Site component.

Domain Warning, site notices and stale cart item notice.

These notices are not relevant to Jetpack Cloud.

This PR removes them in the Jetpack Cloud environment. 

This PR was is inteded to remove the following notice in Jetpack Cloud 
<img width="482" alt="Screen Shot 2020-04-09 at 1 36 29 PM" src="https://user-images.githubusercontent.com/115071/78891100-28072b00-7a67-11ea-9dda-44c052e531cb.png">

#### Testing instructions

* Load this PR in Calypso. 
* Notice the Async calls for domain warnings, notices, and stale card in the network inspector. 

<img width="1327" alt="Screen Shot 2020-04-09 at 1 36 17 PM" src="https://user-images.githubusercontent.com/115071/78891123-305f6600-7a67-11ea-9e83-83b91cc084c4.png">

* Load the PR in Jetpack Cloud. 
* Notice that no async calls are done for the notices in the network inspector in the developer tools.
